src/components/ChatLog.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types'
+import ChatEntry from './ChatEntry';
+
+const ChatLog = ({ entries, onLikeChange }) => {
+    let isLocal = false;
+
+    return (
+        <div className='chat-log'>
+            {
+                entries.map((entry) => {
+                // Toggle the value of isLocal for each entry
+                isLocal = !isLocal;
```

Comment:
Clever way to toggle whether each subsequent message is local or remote. This assumes that the first message is local (toggles to true before passing to the first entry), and that each message alternates after. But what would happen if either sender sent more tha one message in a row?